### ISSUE TITLE
failed to install in windows 11

### ISSUE THREAD

**Issue by NourEldin-Osama** [yobix-ai/extractous#13]
```bash
> pip install extractous 
```
why it's not installing out of the box like other rust based library (for example [uv](https://github.com/astral-sh/uv))
```
Collecting extractous
  Using cached extractous-0.1.4.tar.gz (142 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      
      Cargo, the Rust package manager, is not installed or is not on PATH.
      This package requires Rust and Cargo to compile extensions. Install it through
      the system's package manager or via https://rustup.rs/
      
      Checking for Rust toolchain....
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by nmammeri** [comment]
It looks like you're building it from source. May I ask what OS/system you're running this on?

**Comment by NourEldin-Osama** [comment]
> It looks like you're building it from source. May I ask what OS/system you're running this on?

Windows 11

**Comment by nmammeri** [comment]
At the moment we don't have it built for Windows yet. We have an issue tracking that #1.

**Comment by NourEldin-Osama** [comment]
Is there any estimated time for when it will come to Windows? I hope it's soon. 🙏

**Comment by NourEldin-Osama** [comment]
Also, can you give me tips on how to help with it? I can try to make a pull request, but I don't have any experience with Rust. I will use AI, but I need some guidance on where to start and what the steps are to support Windows. Any help will be appreciated.
Thanks in Advance 🤗

**Comment by NourEldin-Osama** [comment]
Thanks, @nmammeri, for your response.

I hope that it will be done ASAP, as I want to use this in a project I am currently working on to speed up document loading. I got very excited when I saw that this project can increase the speed by up to 25x 🔥 compared to unstructured. I’m waiting for any information about the estimate or the plan.

Thanks in advance for your contribution and effort.

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by nmammeri** [comment]
@NourEldin-Osama please check again, with the help of @KapiWow we have just published a v0.1.5 that has windows support. Waiting for your feedback 😅

**Comment by NourEldin-Osama** [comment]
Hello @nmammeri, I tested now `extractous==0.1.5` on my Windows 11 device and it works 🎉❤️
Thanks for you and for @KapiWow as well
